dynomite-derive/src/attr.rs

```diff
@@ -7,20 +7,36 @@ use syn::{
 };
 
 #[derive(Clone)]
-pub enum Attr {
+pub(crate) struct Attr {
```

Comment:
What was the motivation behind this change?

Reply:
The motivation is to extract the `ident` into a separate field so that accessing it was trivial instead of doing a match:
```rust
let ident = match attr {
    Attr::Default(it) => it,
    Attr::Rename(it, _) => it,
    Attr::PartitionKey(it) => it,
    // ...
};
```

Reply:
Thanks for clarifying